test/unit/bibDetailsUtils.test.js

```diff
@@ -268,11 +268,11 @@ describe('bibDetailsUtils', () => {
 
   describe('stringDirection', () => {
     it('should return \'rtl\' for right-to-left strings', () => {
-      expect(stringDirection('\u200F\u00E9')).to.eql('rtl')
+      expect(stringDirection('\u200F\u00E9', true)).to.eql('rtl')
```

Comment:
This can be a variable `const useParallels = true;` to make it easier to read and know why the `true` value is being passed here.